packages/eds-core-react/stories/playground/DataTable.tsx

```diff
@@ -122,18 +161,20 @@ const MenuButton = ({ row }: { row: string[] }) => {
         aria-haspopup="true"
         aria-expanded={isOpen}
         onClick={(e) => setAnchorEl(e.currentTarget)}
+        onKeyDown={onKeyPress}
       >
         <Icon name="more_vertical" title="more"></Icon>
       </Button>
-      {isOpen && (
+      {isElement && (
         <Menu
           id={`menu-${row.toString()}`}
           aria-labelledby={`menu-button-${row.toString()}`}
-          open={isOpen}
+          open={isElement}
           anchorEl={anchorEl}
-          onClose={() => setAnchorEl(null)}
+          focus={focus}
+          onClose={closeMenu}
         >
-          <Menu.Item>
+          <Menu.Item onKeyDown={onKeyPress} onClick={closeMenu}>
```

Comment:
Menu.item should not have onKeyDown, only onClick. It is a button element so pressing enter on it is same as onClick. This should fix the menu flying around the screen